### DISCUSSION TITLE
Poll: Upcoming Features

### DISCUSSION THREAD

**Discussion by nonsalant** [nonsalant/contract#15] [Polls]
Which of the following 5 features should be worked on first?

**Comment by sicenul** [comment]
I already try the demo. It's better to preview first, before signed and delete the php file. So the client can see/preview the final result before confirming the signed.

**Reply by nonsalant** [reply]
Not sure I agree about the usefulness of having this. Do you know of any other contract signing service that allows the client (not just the creator of the contract) to preview signatures? 🤔

**Comment by nixbitpie** [comment]
Custom (text) fields that a client completes before signing would be nice. Things like their details (name, address, fiscal info, banking info, etc)
